pandas/core/indexes/base.py

```diff
@@ -4902,13 +4903,6 @@ def _searchsorted_monotonic(self, label, side='left'):
 
         raise ValueError('index must be monotonic increasing or decreasing')
 
-    def _get_loc_only_exact_matches(self, key):
```

Comment:
No longer necessary since `get_loc` is now exact for `IntervalIndex`, and was only used with `IntervalIndex`